tests/test_utils.py

```diff
@@ -4,7 +4,10 @@
 
 import unittest
 
-from astroid import Uninferable, builder, nodes
+import pytest as pytest
```

Comment:
```suggestion
import pytest
```

This seems like a nice `pylint` warning to add 😄

Reply:
We have it, it should be raised here. Conf need fixing ?

Reply:
> `Suggested change`

I am using PyCharm and this import was added as "Quick Action", so I missed this strange import code.

> Conf need fixing ?

It seems that these errors are reported in [`Checks job`](https://github.com/PyCQA/astroid/actions/runs/4152907796/jobs/7189573258#step:10:27).
Unfortunately, I have not seen it earlier - maybe the job was not started, because it required an approval.